VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/vho-hearings-filter/vho-hearings-filter.component.html

```diff
@@ -4,46 +4,73 @@
-            <input class="govuk-checkboxes__input" type="checkbox" [checked]="child.selected" id="{{'status' + i}}"
-                   name="{{'status'+i}}" (click)="statusOptionSelected(i)">
-            <label class="govuk-label govuk-checkboxes__label vh-word"
-              for="{{'status' + i}}">{{child.description}}</label>
+            <input
+              class="govuk-checkboxes__input"
+              type="checkbox"
+              [checked]="child.selected"
+              id="{{ 'status' + i }}"
+              name="{{ 'status' + i }}"
+              (click)="statusOptionSelected(i)"
+            />
+            <label class="govuk-label govuk-checkboxes__label vh-word" for="{{ 'status' + i }}">{{ child.description }}</label>
           </div>
         </div>
       </div>
       <div class="govuk-grid-column-one-half">
-
         <p class="govuk-body-l">Alerts</p>
         <div class="govuk-checkboxes__item vh-mb10">
-          <input class="govuk-checkboxes__input" type="checkbox" [checked]="alertsAllChecked" id="alert-all"
-            name="alert-all" (click)="alertAllSelected()">
+          <input
+            class="govuk-checkboxes__input"
+            type="checkbox"
+            [checked]="alertsAllChecked"
+            id="alert-all"
+            name="alert-all"
+            (click)="alertAllSelected()"
+          />
           <label class="govuk-label govuk-checkboxes__label" for="alert-all">All</label>
         </div>
         <hr />
-        <div *ngFor="let child of hearingsFilter.alerts; let i=index">
+        <div *ngFor="let child of hearingsFilter.alerts; let i = index">
           <div class="govuk-checkboxes__item vh-mb10">
-            <input class="govuk-checkboxes__input" type="checkbox" [checked]="child.selected" id="{{'alert' + i}}"
-                   name="{{'alert' + i}}" (click)="alertOptionSelected(i)">
-            <label class="govuk-label govuk-checkboxes__label vh-word"
-              for="{{'alert' + i}}">{{child.description}}</label>
+            <input
+              class="govuk-checkboxes__input"
+              type="checkbox"
+              [checked]="child.selected"
+              id="{{ 'alert' + i }}"
+              name="{{ 'alert' + i }}"
+              (click)="alertOptionSelected(i)"
+            />
+            <label class="govuk-label govuk-checkboxes__label vh-word" for="{{ 'alert' + i }}">{{ child.description }}</label>
           </div>
         </div>
       </div>
     </div>
     <div class="govuk-grid-row">
       <hr />
       <div class="govuk-grid-column-one-half vh-mt20">
-        <input type="button" value="Apply filters" (click)="applyFilters()" id="applyFilter" class="govuk-button"
-          [disabled]="disableFilterApply" />
+        <input
+          type="button"
+          value="Apply filters"
+          (click)="applyFilters()"
+          id="applyFilter"
+          class="govuk-button"
+          [disabled]="disableFilterApply"
+        />
```

Comment:
@shaed-parkar inputs should be inputs, and buttons should be buttons for accessibility reasons. Please can you change it?

Reply:
It is a button. The full element looks like

``` 
<input
type="button"
value="Apply filters"
(click)="applyFilters()"
id="applyFilter"
class="govuk-button"
[disabled]="disableFilterApply"
/>
```

Reply:
no it should be <button> not <input> if its a button

Reply:
Ta